libs/theme/src/ztx.ts

```diff
@@ -0,0 +1,67 @@
+import { CustomTheme } from "./abstract";
+import { common } from "./common";
+
+export const ztxTheme: CustomTheme = {
+  ...common,
+  colors: {
+    primary: "#243FEA",
+    background: "#000000",
+    panelBackground: "#363D4D",
+  },
+  background: {
+    color: "#16151D",
+    backgroundBlendMode: "luminosity",
```

Comment:
Shouldn't that be part of the `welcome` prop? Since that is the one with the problematic image?